src/opencmiss/zincwidgets/sceneeditorwidget.py

```diff
@@ -119,48 +117,59 @@ def _buildGraphicsList(self):
         """
         Fill the graphics list view with the list of graphics for current region/scene
         """
-        if self._graphicsItems is not None:
-            self._graphicsItems.clear()  # Must clear or holds on to graphics references
-        self._graphicsItems = QtGui.QStandardItemModel(self._ui.graphics_listview)
-        selectedIndex = None
+        if self._ui.graphics_listWidget is not None:
+            self._ui.graphics_listWidget.clear()  # Must clear or holds on to graphics references
         if self._scene:
             selectedGraphics = self._ui.graphics_editor.getGraphics()
             graphics = self._scene.getFirstGraphics()
+            index = 0
             while graphics and graphics.isValid():
                 name = self._getGraphicsDisplayName(graphics)
-                item = QtGui.QStandardItem(name)
-                item.setData(graphics)
-                item.setCheckable(True)
-                item.setEditable(False)
+                item = QtWidgets.QListWidgetItem(name)
+                item.setFlags(item.flags() | QtCore.Qt.ItemIsUserCheckable)
                 visible = graphics.getVisibilityFlag()
                 item.setCheckState(QtCore.Qt.Checked if visible else QtCore.Qt.Unchecked)
-                self._graphicsItems.appendRow(item)
                 if graphics == selectedGraphics:
-                    selectedIndex = self._graphicsItems.indexFromItem(item)
+                    self._ui.graphics_listWidget.setCurrentItem(item)
```

Comment:
The new code doesn't select a graphics object when first opened, but the first one is shown in the graphics editor. I'd like the one shown in the graphics editor to be shown as selected in the list at all times.
I can't see why the code isn't working; I would expect setScene() should have passed in the first graphics to the graphics editor before calling this function and therefore it would be selected by this point. Please investigate.